src/session.ts

```diff
@@ -808,7 +808,7 @@ export class SessionManager implements Middleware {
 
             new SessionMenuItem(
                 "Modify 'powerShell.powerShellAdditionalExePaths' in Settings",
```

Comment:
Might be a good opportunity to change the quick pick item text in the session menu too. Any suggestions?

Reply:
Uhh I'm so bad at UX... "Modify list of additional PowerShell executable locations?"

Reply:
```suggestion
                "Modify list of additional PowerShell locations",
```

executable is redundant.

Reply:
done